src/bundle/Controller/ContentTypeController.php

```diff
@@ -504,7 +504,7 @@ public function updateAction(
                 $this->contentTypeActionDispatcher->dispatchFormAction(
                     $form,
                     $form->getData(),
-                    $form->getClickedButton() ? $form->getClickedButton()->getName() : null,
```

Comment:
I would say if this was correct, then instead of disabling submitting by enter key, you should just handle end redirect if button is not clicked.